experiment/measurer/test_coverage_utils.py

```diff
@@ -24,11 +24,12 @@ def get_test_data_path(*subpaths):
     return os.path.join(TEST_DATA_PATH, *subpaths)
 
 
-def test_extract_covered_regions_from_summary_json(fs):
-    """Tests that extract_covered_regions_from_summary_json returns the covered
-    regions from summary json file."""
+def test_extract_covered_branches_from_summary_json(fs):
+    """Tests that extract_covered_branches_from_summary_json returns the covered
+    branches from summary json file."""
     summary_json_file = get_test_data_path('cov_summary.json')
     fs.add_real_file(summary_json_file, read_only=False)
-    covered_regions = coverage_utils.extract_covered_regions_from_summary_json(
+    covered_branches = coverage_utils. \
+    extract_covered_branches_from_summary_json(
         summary_json_file)
-    assert len(covered_regions) == 15
+    assert len(covered_branches) == 9
```

Comment:
I wonder what the effect of branches being less common than regions has on our unique coverage metric. I imagine the unique coverage of various fuzzers will look even smaller than it already does. I guess the relative size wont change but still.